<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Изменение плотности основного
вещества (кг/м^3)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Площадь кристалла (м^2):</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="s_kr_le">
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="mui_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Динамическая вязкость примеси, принимает значения от 0.001 до 0.1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="c0_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Принимает значения от 10^10 до 10^15</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Концентрация примеси в твердом
состоянии (г/л)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Динамическая вязкость (Н*с/м^2)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="ro_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Начальная концентрация
примеси Co (cм^-3):</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Концентрация примеси в жидком
состоянии (г/л)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="yi_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Должен принимать значения от 10^(-5) до 10^(-3)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="15">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="czh_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="d_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Молярная масса компонента (г/моль)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="15">
     <widget class="QGroupBox" name="output_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>485</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>485</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Выходные характеристики:</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Оценка толщины диффузного слоя (мм):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="delta_mean_l">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Эффективный коэффициент:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Коэффициент испарения (только у летучих примесей):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="ki_mean_l">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Общий коэффициент распределения
(только у летучих примесей):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="kob_l">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Обратная задача (характеристики установки):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Диаметр тигля (мм):</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="d_t_l">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="k_mean_l">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Значение</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="ct_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Принимает значения от 10^15 до 10^17</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Скорость кристаллизации f (мм/мин):</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="mu_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>398</width>
        <height>128</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Плотность жидкости (кг/м^3)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="substance_type_cb">
      <item>
       <property name="text">
        <string>Выберите примесь</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B - Бром</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P - Фосфор (летучая)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>As - Мышьяк</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sb - Сурьма (летучая)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="start_process_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Начать процесс моделирования</string>
      </property>
      <property name="statusTip">
       <string>Начать процесс моделирования</string>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Введите параметры смеси:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="f_le">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>178</width>
        <height>22</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Скорость кристаллизации, принимает значения от 0 до 15 мм/мин</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="15">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="zero_page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Тут будут ваши графики и выходные характеристики</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graph_page"/>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Масса в долях (%)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="input_rules_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Моделирование</string>
    </property>
    <addaction name="change_machine_setup_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Полноэкранный режим</string>
     </property>
     <addaction name="set_fullscreen"/>
     <addaction name="disable_fullscreen"/>
    </widget>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="about_action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="change_machine_setup_action">
   <property name="text">
    <string>Изменение параметров установки</string>
   </property>
  </action>
  <action name="set_fullscreen">
   <property name="text">
    <string>Включить</string>
   </property>
  </action>
  <action name="disable_fullscreen">
   <property name="text">
    <string>Выключить</string>
   </property>
  </action>
  <action name="input_rules_action">
   <property name="text">
    <string>Правила ввода</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>substance_type_cb</tabstop>
  <tabstop>f_le</tabstop>
  <tabstop>c0_le</tabstop>
  <tabstop>mu_le</tabstop>
  <tabstop>ro_le</tabstop>
  <tabstop>ct_le</tabstop>
  <tabstop>czh_le</tabstop>
  <tabstop>d_le</tabstop>
  <tabstop>mui_le</tabstop>
  <tabstop>yi_le</tabstop>
  <tabstop>s_kr_le</tabstop>
  <tabstop>start_process_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
